scripts/welcome.js

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
+/* eslint-disable no-console */
+const prompts = require("prompts");
+const dedent = require("ts-dedent").default;
+const path = require("path");
+const fs = require("fs");
+const { execSync } = require("child_process");
+
+// CLI questions
+const questions = [
+  {
+    type: "text",
+    name: "authorName",
+    initial: "",
+    message: "What is the package author name?*",
+    validate: (name) => (name === "" ? "Name can't be empty" : true),
+  },
+  {
+    type: "text",
+    name: "authorEmail",
+    initial: "",
+    message: "What is the package author email?",
+  },
+  {
+    type: "text",
+    name: "packageName",
+    message: "What is the addon package name (eg: storybook-addon-something)?*",
+    validate: (name) => (name === "" ? "Package name can't be empty" : true),
+  },
+  {
+    type: "text",
+    name: "displayName",
+    message: "What is the addon display name?*",
```

Comment:
Maybe add an example or further describe what that means? As I user I could write Storybook Addon Interactions, but in the README will become Storybook Storybook Addon